### PR TITLE
Info ver

### PR DESCRIPTION
**Pull request by Arturoo0**

### CODE REVIEW

File: ftpcontroller.py [hunk 1]
```diff
@@ -14,9 +14,12 @@ def __init__(self, host, port=ftputils.FTP_PORT):
...
     self.commandSocket = ftputils.getTCPSocket()
-    self.commandSocket.connect((self.commandHost, self.commandPort))
-    self.commandSocketFile = self.commandSocket.makefile('r')
-    self.appendToBuffer(f'Successfully connected to {self.commandHost}.')
+    try:
+        self.commandSocket.connect((self.commandHost, self.commandPort))
+        self.commandSocketFile = self.commandSocket.makefile('r')
+        self.appendToBuffer(f'Successfully connected to {self.commandHost}.')
+    except Exception:
+        sys.exit('Something went wrong connecting to host \'{}\'.'.format(self.commandHost))
```
Comment (Bruception) [inline ftpcontroller.py]: Can we just use a literal f string here?  f’’

File: ftpcontroller.py [hunk 2]
```diff
@@ -72,7 +75,13 @@ def readData(self, command, argument, dataSocket):
 
   # TODO Handle missing file
   def sendData(self, argument, dataSocket):
-    sourceFile = open(argument, 'rb')
+    try:
+      sourceFile = open(argument, 'rb')
+    except FileNotFoundError:
+      dataSocket.close()
+      self.dumpResponseBuffer()
+      self.appendToBuffer('File \'{}\' not found within current directory.'.format(argument))
```
Comment (Bruception) [inline ftpcontroller.py]: We aren’t sending the ABOR command? Also we should return the same message that we are adding to the buffer.

File: ftputils.py [hunk 3]
```diff
@@ -91,7 +92,10 @@ def getTCPSocket():
   return socket.socket(socket.AF_INET, socket.SOCK_STREAM)
 
 def getFTPLine():
-  line = input('ftp> ')
+  try:
+    line = input('ftp> ')
+  except KeyboardInterrupt:
```
Comment (Bruception) [inline ftputils.py:97]: Nice :)

**Review by Bruception — CHANGES_REQUESTED** [review]

File: ftpcontroller.py (hunk 1 above)
Comment (Arturoo0) [reply ftpcontroller.py:22]: nah

**Review by Bruception — CHANGES_REQUESTED** [review]
LGTM! Just update the README and remove the TODOS in the code.

**Review by Bruception — APPROVED** [review]
LGTM!

**Pull request merged**